modules/core/src/main/java/org/apache/fluo/core/impl/TransactionImpl.java

```diff
@@ -190,6 +190,8 @@ public Bytes get(Bytes row, Column column) {
     return ret;
   }
 
+  // TODO: Fix override to return Map<RowColumn, Bytes>
+  // TODO: Fix span() as this will move to unsupported type.
```

Comment:
Could create a new `HashMap<RowColumn, Bytes>` and loop over what pss.scan() returned adding to the new hash map.   Could add an inner loop to exisitng loop.... like the following.

```java
//create HashMap<RowColumn, Bytes>

for (Entry<Bytes, Map<Column, Bytes>> entry : ret.entrySet()) {
   updateColumnsRead(entry.getKey(), entry.getValue().keySet());
   for (Entry<Column, Bytes> cols : entry.getValue().entrySet()) {
    //create RowColumn  and add to new map
   }    
}

//return new map
```